.github/workflows/ci.yml

```diff
@@ -134,25 +134,6 @@ jobs:
         run: yarn stylelint
         working-directory: packages/website
 
-  eslint_v9_tests:
```

Comment:
[Testing] We'll still want to test against ESLint 8. I don't think anything would now do that? Unless I'm missing something, this should be switched to `eslint_v8_tests`, not removed.

Reply:
We still do have [this integration test for ESLint 8](https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/integration-tests/fixtures/eslint-v8).

@bradzacher's [thoughts from above](https://github.com/typescript-eslint/typescript-eslint/pull/9119#issuecomment-2124749156)

and looking at the history of this file, it seems like we have not had similar tests in the past for other versions of ESLint

Reply:
I don't think we need to worry too much.
There aren't any new APIs we need to worry about for plugins at least.
Given we have extension rules we need to be careful testing across majors. I'm surprised none of our tests broke!

When we had v7+v8 support we only relied on the integration test and not a full CI.
I think this might just be good enough.

Reply:
Ah, great, this is enough for me. Thanks! 🎉